staging/src/k8s.io/legacy-cloud-providers/azure/clients/armclient/azure_armclient.go

```diff
@@ -135,19 +135,19 @@ func (c *Client) Send(ctx context.Context, request *http.Request) (*http.Respons
 	}()
 
 	bodyString := string(bodyBytes)
-	klog.V(5).Infof("Send.sendRequest original error message: %s", bodyString)
+	klog.V(5).InfoS("Send.sendRequest original error message", "responseBody", bodyString)
 
 	// Hack: retry the regional ARM endpoint in case of ARM traffic split and arm resource group replication is too slow
 	var body map[string]interface{}
 	if e := json.Unmarshal(bodyBytes, &body); e != nil {
```

Comment:
If you're up for it, it'd be great to change the `e` to `err` to make it clearer